coral/src/domain/schema-request/schema-request-transformer.test.ts

```diff
@@ -31,6 +31,7 @@ describe("schema-request-transformer.ts", () => {
           appname: "App",
           schemafull: "",
           username: "jlpicard",
+          requestor: "jlpicard",
           requesttime: "1987-09-28T13:37:00.001+00:00",
```

Comment:
kjaneway is missing

Reply:
`bsisko` though

Reply:
I couldn't very well add Sisko or Janeway to  team `NCC1701D` 😱